test/remote/gateways/remote_sum_up_test.rb

```diff
@@ -123,6 +123,26 @@ def test_failed_void_invalid_checkout_id
     assert_equal 'Resource not found', response.message
   end
 
+  def test_successful_refund
+    gateway = SumUpGateway.new(fixtures(:sum_up_testing))
+    purchase = gateway.purchase(@amount, @credit_card, @options)
+    assert_success purchase
+    assert_equal 'PAID', purchase.message
```

Comment:
💬 Comment:

Do we need all those assertions on the part of the code that is just the context/preparation for the actual test that is going to be `refund`?, there are 7 assertions before the refund request is actually fired and that request only has 2 assertions, are we testing `purchase` or `refund`?